tools/tpm2_pkcs11/tpm2.py

```diff
@@ -53,6 +52,18 @@ def evictcontrol(ownerauth, ctx):
 
     def load(self, pctx, pauth, priv, pub):
 
+        if isinstance(priv, buffer):
+            sealprivf = NamedTemporaryFile()
+            sealprivf.write(priv)
+            sealprivf.flush()
+            priv = sealprivf.name
+
+        if isinstance(pub, buffer):
```

Comment:
Same as above

Reply:
@diabonas so glad you added the PYTHON3 tests into the CI system. I'm liking that were failing up-front!